datatable_tutorial/datatable_tutorial/datatable_tutorial.py

```diff
@@ -0,0 +1,269 @@
+    "headerFontStyle": "bold 14px",
+    "baseFontStyle": "13px",
+    "fontFamily": "Inter, Roboto, -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Ubuntu, noto, arial, sans-serif",
+}
+
+def index() -> rx.Component:
+    return rx.fragment(
+        rx.color_mode_button(rx.color_mode_icon(), float="right"),
+        rx.vstack(
+            rx.heading("Data Table Demo!", font_size="2em"),
+            rx.vstack(
+                rx.button("Static / Live Data", on_click=DataTableState.change, color_scheme='messenger', size='lg',),
+                rx.box(height="1em"),
+                rx.cond(
+                    DataTableState.show,
+                    rx.vstack(
+                        rx.heading(DataTableState.clicked_cell, size="lg", color="blue"),
+                        rx.heading(DataTableState.edited_cell, size="lg", color="green"),
+                        rx.heading(DataTableState.right_clicked_group_header, size="lg", color="orange"),
+                        rx.heading(DataTableState.item_hovered, size="lg", color="purple"),
+                        rx.heading(DataTableState.deleted, size="lg", color="red"),
+                        rx.data_editor(
+                            columns=DataTableState.cols,
+                            data=DataTableState.data,
+
+                            #rows=10,
+                            on_paste=True,
+                            draw_focus_ring=False,
+                            #fixed_shadow_x=True,
+
+                            
+                            freeze_columns=2,
+                            group_header_height=100,
+                            header_height=80,
+                            #max_column_auto_width=200,
+                            # this works just need to describe it
+                            #max_column_width=200,
+                            min_column_width=100,
+                            row_height=50,
+                            row_markers='clickable-number',
+                            # also mention smooth_scroll_y
+                            smooth_scroll_x=True,
+                            vertical_border=False,
+                            column_select="multi",
+                            #prevent_diagonal_scrolling=False,
+                            overscroll_x=0,
+
+
+                            on_cell_clicked=DataTableState.get_clicked_data,
+                            on_cell_edited=DataTableState.get_edited_data,
+                            on_group_header_context_menu=DataTableState.get_group_header_right_click,
+                            on_item_hovered=DataTableState.get_item_hovered,
+                            on_delete=DataTableState.get_deleted_item,
+                            #on_row_appended=DataTableState.append_row,
+                            on_column_resize=DataTableState.column_resize,
+                            theme=darkTheme,
```

Comment:
would be cool to use something like

```python
theme=rx.color_mode_cond(light={}, dark=darkTheme),
```

but apparently this is broken in the latest version of reflex (curly braces are stripped out of cond props)

Reply:
let's try and add this in after when it is working again